opentelemetry-api/tests/context/test_contextvars_context.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+from unittest.mock import patch
+
+from opentelemetry import context
+
+try:
+    import contextvars  # pylint: disable=unused-import
+    from opentelemetry.context.contextvars_context import (
+        ContextVarsRuntimeContext,
+    )
+except ImportError:
+    raise unittest.SkipTest("contextvars not available")
+
+
+def do_work() -> None:
+    context.set_current(context.set_value("say", "bar"))
+
+
+class TestContextVarsContext(unittest.TestCase):
+    def setUp(self):
+        self.previous_context = context.get_current()
+
+    def tearDown(self):
+        context.set_current(self.previous_context)
+
+    @patch(
+        "opentelemetry.context._RUNTIME_CONTEXT", ContextVarsRuntimeContext()  # type: ignore
```

Comment:
I'm not sure why the CLI `black` isn't wrapping these. FWIW [`black.format_file_contents` via the vim plugin](https://github.com/psf/black/blob/master/plugin/black.vim#L119) does.

Reply:
It could be line length isn't being honored. I see it in the pyproject.toml, but IIRC my editor wasn't always honoring that configuration either.